packages/styles/scss/components/card-in-card/_card-in-card.scss

```
@@ -98,6 +102,7 @@
       opacity: 0;
       z-index: 1;
       background-color: $inverse-02;
+      transition: all 150ms cubic-bezier(0.2, 0, 0.38, 0.9) 0s;
```

We should be leveraging tokens from [Carbon motion guidelines](https://www.carbondesignsystem.com/guidelines/motion/overview/) for our transitions. Also, any chance we could get more selective with our transitions? Helps a little with debugging, but also performance to my understanding.

```suggestion
      transition: opacity $moderate-01 motion('standard', 'productive');
```

---

Agreed, targeting `all` is known to reduce performance.